### PR TITLE
Give decent error message if model not found

### PR DESCRIPTION
**Pull request by rhatdan**
Rename podman_build.sh to build.sh

### CODE REVIEW

File: ci.sh [hunk 1]
```diff
@@ -11,14 +11,14 @@ main() {
     apt install -y python3-autopep8
   fi
 
-  ./podman-build.sh
```
Comment (ericcurtin) [inline ci.sh:14]: The thing is about this is, it doesn't build ramalama, it only builds container images...

Maybe container-build.sh ?

Not a blocking issue for this PR anyway

Comment (ericcurtin) [reply ci.sh:14]: We can catch in a follow on PR if needs be

File: ci.sh [hunk 2]
```diff
@@ -7,18 +7,22 @@ available() {
 main() {
   set -ex -o pipefail
 
-  if ! available autopep8 && available apt; then
-    apt install -y python3-autopep8
+  if ! available autopep8; then
```
Comment (ericcurtin) [inline ci.sh:10]: I already fixed the sudo thing, it's one of the reasons we have conflicts in this PR

**Review by ericcurtin — APPROVED** [review]
LGTM, waiting on green build

**Comment by rhatdan** [comment]
@ericcurtin can we use the small model for ci rather then granite?

**Comment by ericcurtin** [comment]
We just need a chmod a+x or whatever to fix CI here I think, assuming python style is autopep8 auto-formatted, etc.

**Comment by ericcurtin** [comment]
We can do the moondream change is this PR if needs be too

**Comment by ericcurtin** [comment]
"tinyllama" is even smaller

https://ollama.com/library/tinyllama

**Pull request merged**